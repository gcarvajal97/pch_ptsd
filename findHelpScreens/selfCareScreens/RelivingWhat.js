import React, { Component } from 'react';
import { View, StyleSheet, Text, ScrollView,} from 'react-native';

class RelivingWhat extends Component {

  render() {
    return (
            <View>

                <Text style={styles.paragraphTitle}>
                    "I think a lot about what happened to my child."
                </Text>
                <Text style={styles.paragraphBold}>
                    Why it happens:
                </Text>
                <Text style={styles.paragraph}>
                    Thinking and talking about what happened, especially in the first 
                    few days and weeks, is part of how we help ourselves recover from a scary experience 
                    or traumatic event.
                </Text>
                <Text style={styles.paragraphBold}>
                    Is this a problem?:
                </Text>
                <Text style={styles.paragraph}>
                    Do thoughts about the traumatic event often pop into your mind 
                    even when you don't want them to? Do the thoughts bother or upset you?
                </Text>
                
                <Text style={styles.paragraphTitle}>
                    "I have had bad dreams or nightmares since the injury."
                </Text>
                <Text style={styles.paragraphBold}>
                    Why it happens:
                </Text>
                <Text style={styles.paragraph}>
                    In a stressful time, it's not unusual to have a few bad dreams.                
                </Text>
                <Text style={styles.paragraphBold}>
                    Is this a problem?:
                </Text>  
                <Text style={styles.paragraph}>
                    Are the nightmares interfering with getting a good night's sleep?
                </Text>
                
                <Text style={styles.paragraphTitle}>
                    "I get upset or have physical symptoms (headache, stomachache, heart beating fast) 
                    at reminders of what happened."
                </Text> 
                <Text style={styles.paragraphBold}>
                    Why it happens:
                </Text>
                <Text style={styles.paragraph}>
                    When something scary happens, we learn to treat things connected 
                    to it as if they were danger signals. Sights, sounds, and even smells can be reminders.
                </Text> 
                <Text style={styles.paragraphBold}>
                    Is this a problem?:
                </Text>  
                <Text style={styles.paragraph}>
                    Do these feelings happen nearly every day? Do they make it hard 
                    for you to relax, or to concentrate on things you have to do at home or at work?
                </Text>             
                
            </View>
    );
  }
}

export default RelivingWhat;

const styles = StyleSheet.create({
    paragraphTitle: {
      fontWeight: 'bold',
      fontSize: 20,
      alignSelf: 'stretch',
      textAlign: 'left',
      marginTop: 6,
      marginBottom: 3,
      color: '#2089DC'
    },
    paragraph: {
      textAlign: 'left',
      fontSize: 14,
      marginVertical: 2,
    },
    paragraphBold: {
      textAlign: 'left',
      fontSize: 14,
      fontWeight: 'bold',
      marginVertical: 2,
      marginLeft: 10
    }
  });