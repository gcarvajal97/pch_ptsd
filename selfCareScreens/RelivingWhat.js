import React, { Component } from 'react';
import { View, StyleSheet, Text, ScrollView,} from 'react-native';

class RelivingWhat extends Component {

  render() {
    return (
            <Text>

                <Text style={styles.subTitle}>
                    "I think a lot about what happened to my child."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    Why it happens: {"\n"}Thinking and talking about what happened, especially in the first 
                    few days and weeks, is part of how we help ourselves recover from a scary experience 
                    or traumatic event.{"\n"}
                    Is this a problem?: {"\n"}Do thoughts about the traumatic event often pop into your mind 
                    even when you don't want them to? Do the thoughts bother or upset you?{"\n"}
                </Text>
                <Text style={styles.subTitle}>
                    "I have had bad dreams or nightmares since the injury."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    Why it happens: {"\n"}In a stressful time, it's not unusual to have a few bad dreams.{"\n"}
                    Is this a problem?: {"\n"}Are the nightmares interfering with getting a good night's sleep?{"\n"}
                </Text>
                <Text style={styles.subTitle}>
                    "I get upset or have physical symptoms (headache, stomachache, heart beating fast) 
                    at reminders of what happened."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    Why it happens: {"\n"}When something scary happens, we learn to treat things connected 
                    to it as if they were danger signals. Sights, sounds, and even smells can be reminders.{"\n"}
                    Is this a problem?: {"\n"}Do these feelings happen nearly every day? Do they make it hard 
                    for you to relax, or to concentrate on things you have to do at home or at work?
                </Text>
                
            </Text>
    );
  }
}

export default RelivingWhat;

const styles = StyleSheet.create({
  container: {
      flex: 1,
      alignItems: 'center',
      margin: 10,
      backgroundColor: 'white',
      shadowColor: "#000",
      shadowOffset: {
          width: 0,
          height: 2,
      },
      shadowOpacity: 0.23,
      shadowRadius: 2.62,
      elevation: 4,
  },
  title: {
      backgroundColor: 'white', 
      color: '#2089DC', 
      fontWeight: 'bold', 
      fontSize: 22, 
      textAlign: 'left', 
      alignSelf: 'stretch'
  },
  subTitle: {
    backgroundColor: 'white', 
    color: '#2089DC', 
    alignSelf: 'stretch', 
    paddingLeft: 15, 
    paddingRight:15, 
    fontSize: 15, 
    borderBottomWidth:2, 
    borderBottomColor:'#2089DC',
  },
  paragraph: {
      alignSelf:"center", 
      paddingVertical:1, 
      paddingHorizontal:5,
      fontSize: 14,
      marginTop: 5,
  },
});