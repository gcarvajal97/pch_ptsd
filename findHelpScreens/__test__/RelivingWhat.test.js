import React from 'react';
import RelivingWhat from '../selfCareScreens/RelivingWhat';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<RelivingWhat/>).toJSON();
    expect(tree).toMatchSnapshot();
}) 