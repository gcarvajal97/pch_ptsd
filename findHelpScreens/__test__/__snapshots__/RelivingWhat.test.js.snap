// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I think a lot about what happened to my child."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Thinking and talking about what happened, especially in the first few days and weeks, is part of how we help ourselves recover from a scary experience or traumatic event.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Do thoughts about the traumatic event often pop into your mind even when you don't want them to? Do the thoughts bother or upset you?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I have had bad dreams or nightmares since the injury."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    In a stressful time, it's not unusual to have a few bad dreams.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Are the nightmares interfering with getting a good night's sleep?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I get upset or have physical symptoms (headache, stomachache, heart beating fast) at reminders of what happened."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    When something scary happens, we learn to treat things connected to it as if they were danger signals. Sights, sounds, and even smells can be reminders.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Do these feelings happen nearly every day? Do they make it hard for you to relax, or to concentrate on things you have to do at home or at work?
  </Text>
</View>
`;
